docs/getting-started/using-the-block-explorer.md

```diff
@@ -4,59 +4,69 @@
 
 ## ![](../img/dcr-icons/Info.svg){ .dcr-icon } Overview
 
-All blocks and transactions on the Decred blockchain
-are visible through the use of the block explorer, [dcrdata](https://github.com/decred/dcrdata).
+All blocks and transactions on the Decred blockchain are visible through the
+use of the block explorer, [dcrdata](https://github.com/decred/dcrdata).
 
 Public instances of dcrdata are available for the following networks:
 
 - [mainnet](https://dcrdata.decred.org)
 - [testnet](https://testnet.decred.org)
 
-Below is a quick review of some of the information on them.
+---
+
+## ![](../img/dcr-icons/Blocks.svg){ .dcr-icon } Blocks
+
+Blocks can be found by searching for their block height number, clicking on a
+`Height` value from the home page, or from their `BlockHash` value. Older
+blocks will have lower block numbers.
 
-Option         | Explanation
----            | ---
-`Height`       | The block number.
-`Age`          | How long ago the block was added to the blockchain.
-`Transactions` | The number of transactions included in the block.
-`Votes`        | The number of proof-of-stake votes included in the block.
-`Fresh Stake`  | The number of new tickets purchased in this block.
-`Size`         | The size (in bytes) of the block.
+Each block is packed with information. There are two main sections inside a
+block: 
 
-Under `Latest Transactions`, you can see the transaction ID (txid) and
-the value (in DCR) transmitted across the network.
+1. **The Block Header**
+    - This shows summary information about this specific block, as well as a
+    link to the previous block in the chain.
+2. **Transactions**
+    - Consisting of:
+        - The Proof-of-Work Reward
+        - The Treasury Reward
+        - Proof-of-Stake Votes and Rewards
+        - Tickets which will enter the stakepool
+        - All other transactions about how DCR is transmitted across the network
 
 ---
 
-## ![](../img/dcr-icons/Blocks.svg){ .dcr-icon } Blocks
+## Block Headers
 
-Blocks can be found by searching for their block height number,
-clicking on a `Height` value from the home page, or from their
-`BlockHash` value. Older blocks will have lower block numbers. The top
-half of a block overview shows relevant information about this
-specific block. This information includes: the block height, the block
-hash, and several key network parameters, described below:
+The following table should help explain what you'll see in the top section of
+[dcrdata](https://dcrdata.decred.org). This is generally all information that's
+also found inside a block header, although it adds some additional context.
 
-Option                   | Explanation
+Parameter                | Explanation
```

Comment:
"Option" was definitely the wrong word here, but I don't particularly like "Parameter" either. To a developer it implies an argument passed into a function (which isn't accurate), and non-developers might not know the word at all. 

How about "Value" instead?